tests/common/devices.py

```diff
@@ -1554,7 +1539,26 @@ def __init__(self, sonichost, asic_index):
         """
         self.sonichost = sonichost
         self.asic_index = asic_index
+        if self.sonichost.is_multi_asic:
+            self.namespace = "{}{}".format(NAMESPACE_PREFIX, self.asic_index)
+        else:
+            # set the namespace to DEFAULT_NAMESPACE(None) for single asic
+            self.namespace = DEFAULT_NAMESPACE
```

Comment:
Would making it "None" work better since ansible modules test for "None" namespace.

Reply:
DEFAULT_NAMESPACE is  None